.github/workflows/slsa3_builder.yml

```diff
@@ -410,3 +418,60 @@ jobs:
           path: "${{ steps.sign-prov.outputs.signed-provenance-name }}"
           if-no-files-found: error
           retention-days: 5
+
+###################################################################
+#                                                                 #
+#           Upload binaries and provenances as assets             #
+#                                                                 #
+###################################################################
+  upload-assets:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    needs: [build,build-dry,provenance]
+    if: startsWith(github.ref, 'refs/tags/') && ${{ inputs.upload-assets == 'true' }}
```

Comment:
this line is not working. I'll troubleshoot that. But the rest is ready for review.

Reply:
Fixed.